kotlin-vercel-ncc/src/jsMain/kotlin/vercel/ncc.kt

```diff
@@ -0,0 +1,12 @@
+package vercel
+
+import kotlinx.coroutines.await
+import kotlin.js.Promise
+
+@JsModule("@vercel/ncc")
+@JsNonModule
```

Comment:
`@JsNonModule` is redundant

Reply:
Hm, indeed.
At some point the compiler told me to add it, but I guess I changed some things that makes it obsolete now.